File: backend-SolicitudPermiso/Models/PermisosModel.cs
using System;
using System.Collections.Generic;
namespace backend_SolicitudPermiso.Models
{
    public class PermisosModel
    {
        public int id {get;set;}
        public int idEmpleado {get;set;}
        public int idTipoPermiso {get;set;}
        public DateTime fechaHoraInicioPermiso {get;set;}
        public DateTime fechaHoraFinPermiso {get;set;}
        public string descripcion {get;set;}
    }

    public class ListadoPermisosModel
    {
        public int id {get;set;}
        public string nombreEmpleado {get;set;}
        public string apellidosEmpleado {get;set;}
        public string tipoPermiso {get;set;}
        public DateTime fechaHoraInicioPermiso {get;set;}
        public DateTime fechaHoraFinPermiso {get;set;}
        public int total {get;set;}
    }

    public class ListadoPermisoEmpleadoModel
    {
        public string nombreEmpleado {get;set;}
        public string apellidosEmpleado {get;set;}
        public int cantPermiso {get;set;}
        public int total {get;set;}
    }

    public class ListadoPermisoTipoPermisoModel
    {
        public string nombreEmpleado {get;set;}
        public string apellidosEmpleado {get;set;}
        public string tipoPermiso {get;set;}
        public int total {get;set;}
    }
}

File: frontend-SolicitudPermiso/src/app/pages/permisos/model/permisos.model.ts
export interface PermisosModel{
    id: number;
    idEmpleado: number;
    idTipoPermiso: number;
    fechaHoraInicioPermiso: string;
    fechaHoraFinPermiso: string;
}

export interface ListadoPermisosModel{
    id: number;
    nombreEmpleado: string;
    apellidosEmpleado: string;
    tipoPermiso: string;
    fechaHoraInicioPermiso: string;
    fechaHoraFinPermiso: string;
    total: number;
}

export interface ListadoPermisoEmpleadoModel{
    nombreEmpleado: string;
    apellidosEmpleado: string;
    cantPermiso: string;
    total: number;
}

export interface ListadoPermisoTipoPermisoModel{
    nombreEmpleado: string;
    apellidosEmpleado: string;
    tipoPermiso: string;
    total: number;
}